packages/http-specs/specs/streaming/jsonl/main.tsp

```diff
@@ -0,0 +1,33 @@
+import "@typespec/http";
+import "@typespec/http/streams";
+import "@typespec/spector";
+
+using TypeSpec.Http;
+using TypeSpec.Http.Streams;
+using Spector;
+
+@doc("Test of jsonl streaming.")
+@scenarioService("/streaming/jsonl")
+namespace Streaming.Jsonl;
+
+@route("basic")
+namespace Basic {
```

Comment:
only potential suggestion would name this `NoStreaming` or something like that so it reflect that this is a case where you get the log ine one go at the end

Reply:
renamed.